### PR TITLE
create_ap: tweak casing

### PR DESCRIPTION
**Pull request by sbrl**
Follow-up to #2884

### CODE REVIEW

**Comment by pxgamer** [comment]
This seems fine to me, I was just thinking though, should we use `my` in the tokens? We usually try to keep things without pronouns. :thinking:

File: pages/linux/create_ap.md [hunk 1]
```diff
@@ -4,28 +4,28 @@
 
 - Create an open network with no passphrase:
 
-`create_ap {{wlan0}} {{eth0}} {{MyAccessPoint}}`
+`create_ap {{wlan0}} {{eth0}} {{my_access_point}}`
```
Comment (mebeim) [inline pages/linux/create_ap.md]:
```suggestion
`create_ap {{wlan0}} {{eth0}} {{access_point_ssid}}`
```

File: pages/linux/create_ap.md [hunk 2]
```diff
@@ -4,28 +4,28 @@
...
 
 - Use a WPA + WPA2 passphrase:
 
-`create_ap {{wlan0}} {{eth0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap {{wlan0}} {{eth0}} {{my_access_point}} {{my_passphrase}}`
```
Comment (mebeim) [inline pages/linux/create_ap.md]:
```suggestion
`create_ap {{wlan0}} {{eth0}} {{access_point_ssid}} {{passphrase}}`
```

File: pages/linux/create_ap.md [hunk 3]
```diff
@@ -4,28 +4,28 @@
...
 
 - Create an access point without Internet sharing:
 
-`create_ap -n {{wlan0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap -n {{wlan0}} {{my_access_point}} {{my_passphrase}}`
```
Comment (mebeim) [inline pages/linux/create_ap.md]:
```suggestion
`create_ap -n {{wlan0}} {{acces_point_ssid}} {{passphrase}}`
```

File: pages/linux/create_ap.md [hunk 4]
```diff
@@ -4,28 +4,28 @@
...
 
 - Create a bridged network with Internet sharing:
 
-`create_ap -m bridge {{wlan0}} {{eth0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap -m bridge {{wlan0}} {{eth0}} {{my_access_point}} {{my_passphrase}}`
```
Comment (mebeim) [inline pages/linux/create_ap.md]:
```suggestion
`create_ap -m bridge {{wlan0}} {{eth0}} {{access_point_ssid}} {{passphrase}}`
```

File: pages/linux/create_ap.md [hunk 5]
```diff
@@ -4,28 +4,28 @@
...
 
 - Create a bridged network with Internet sharing and a pre-configured bridge interface:
 
-`create_ap -m bridge {{wlan0}} {{br0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap -m bridge {{wlan0}} {{br0}} {{my_accesspoint}} {{my_passphrase}}`
```
Comment (mebeim) [inline pages/linux/create_ap.md]:
```suggestion
`create_ap -m bridge {{wlan0}} {{br0}} {{access_point_ssid}} {{passphrase}}`
```

File: pages/linux/create_ap.md [hunk 6]
```diff
@@ -4,28 +4,28 @@
...
 
 - Create an access port for Internet sharing from the same WiFi interface:
 
-`create_ap {{wlan0}} {{wlan0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap {{wlan0}} {{wlan0}} {{my_access_point}} {{my_passphrase}}`
```
Comment (mebeim) [inline pages/linux/create_ap.md]:
```suggestion
`create_ap {{wlan0}} {{wlan0}} {{access_point_ssid}} {{passphrase}}`
```

File: pages/linux/create_ap.md [hunk 7]
```diff
@@ -4,28 +4,28 @@
...
 
 - Choose a different WiFi adapter driver:
 
-`create_ap --driver {{WifiAdapter}} {{wlan0}} {{eth0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap --driver {{WifiAdapter}} {{wlan0}} {{eth0}} {{my_access_point}} {{my_passphrase}}`
```
Comment (mebeim) [inline pages/linux/create_ap.md]:
```suggestion
`create_ap --driver {{WifiAdapter}} {{wlan0}} {{eth0}} {{access_point_ssid}} {{passphrase}}`
```

File: pages/linux/create_ap.md [hunk 8]
```diff
@@ -4,28 +4,28 @@
...
 
 - Create an access port for Internet sharing from the same WiFi interface:
 
-`create_ap {{wlan0}} {{wlan0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap {{wlan0}} {{wlan0}} {{access_point_ssid}} {{passphrase}}`
 
 - Choose a different WiFi adapter driver:
 
-`create_ap --driver {{WifiAdapter}} {{wlan0}} {{eth0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap --driver {{WifiAdapter}} {{wlan0}} {{eth0}} {{access_point_ssid}} {{passphrase}}`
```
Comment (schneiderl) [inline pages/linux/create_ap.md]: Isn't it better if we go for something like this just like we are doing on other tokens?
```suggestion
`create_ap --driver {{wifi_adapter}} {{wlan0}} {{eth0}} {{access_point_ssid}} {{passphrase}}`
```

Comment (pxgamer) [reply pages/linux/create_ap.md]: Yes, I'd agree with that. Hadn't even noticed it. 👍

Comment (mebeim) [reply pages/linux/create_ap.md]: Oh, I missed that too! Nice catch :)

**Pull request merged**